config/software/procps-ng.rb

```diff
@@ -16,13 +16,21 @@
 
 build do
   ship_license "https://gitlab.com/procps-ng/procps/raw/master/COPYING"
+
+  # By default procps-ng will build with the 'UNKNOWN' version if not built
```

Comment:
The toolchain changes to the builder are necessary regardless right? We need to update autoconf et al, to build `3.3.16` correct? It's not just related to the 'UNKNOWN' version and `autogen.sh` right?

Reply:
cc @hush-hush 

(merging the PR as-is now, in time for the freeze)

Reply:
@truthbk Yes the change to the builder are here only to be able to run the `autogen.sh` (which we use to run locally and upload the result to our internal S3).

This comment is just to explain why we write the version to that file.